From 3c46b61edb9054143c255e3a3d74298ea3c758c7 Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Tue, 11 Oct 2011 01:40:12 +0530
Subject: [PATCH 0610/1092] arm:omap3:am33xx: HACK for bypassing RTC hwmod

Temp patch to avoid rtc clock getting reset
Needs to be investigated

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/mach-omap2/omap_hwmod_33xx_data.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
index c8fe433..5663d13 100644
--- a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
+++ b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
@@ -2524,10 +2524,6 @@ static __initdata struct omap_hwmod *am33xx_hwmods[] = {
 	&am33xx_ocmcram_hwmod,
 	/* ocpwp class */
 	&am33xx_ocpwp_hwmod,
-	/* rtc class */
-#if 0
-	&am33xx_rtc_hwmod,
-#endif
 	/* sha0 class */
 	&am33xx_sha0_hwmod,
 	/* smartreflex class */
-- 
1.7.11.2

